diff --git a/gwdetchar/tests/test_daq.py b/gwdetchar/tests/test_daq.py
index c55d14c..2951196 100644
--- a/gwdetchar/tests/test_daq.py
+++ b/gwdetchar/tests/test_daq.py
@@ -34,7 +34,6 @@ from gwpy.segments import (Segment, SegmentList, DataQualityDict)
 from gwpy.testing.utils import assert_segmentlist_equal

 from .. import daq
-from ..io import ligolw

 OVERFLOW_SERIES = TimeSeries([0, 0, 0, 1, 1, 0, 0, 1, 0, 1], dx=.5)
 CUMULATIVE_SERIES = TimeSeries([0, 0, 0, 1, 2, 2, 2, 3, 3, 4], dx=.5)
